rules-java/api/pom.xml

```diff
@@ -35,6 +35,23 @@
             <version>5.0.4</version>
         </dependency>
 
+        <!-- Local Dependencies -->
+        <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data</artifactId>
+            <version>6-SNAPSHOT</version>
+            <scope>compile</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-core</artifactId>
+            <version>4.8.1</version>
```

Comment:
We don't put external dependencies like that into the bom. Although it does seem like this could be pulled in from graph-api which already exposes lucene capabilities to the rest of Windup.